src/mono/wasm/wasm.targets

```diff
@@ -63,7 +63,7 @@
     <Copy SourceFiles="$(NativeBinDir)dotnet.js;
                        $(NativeBinDir)dotnet.wasm;
                        $(NativeBinDir)dotnet.timezones.blat;
-                       $(NativeBinDir)icudt.dat"
+                       $(NativeBinDir)icudt_optimal.dat"
```

Comment:
Should we add a property that can be set for the filter name and then copy the according .dat file?

Reply:
@safern ah, indeed, actually it should copy all of the *.dat files to the runtime -pack
fixed.